commands/publish/__tests__/command.test.js

```diff
@@ -0,0 +1,23 @@
+"use strict";
+
+const command = require("../command");
+const yargs = require("yargs");
```

Comment:
We're not testing yargs, please remove this file. The test added to publish-command.test.js is sufficient to verify the option config was recognized.

Reply:
- [ ] Remove useless file